docs/integrations/integration-basics/ingestion-modes.md

```diff
@@ -1,12 +1,22 @@
+---
+title: Ingestion Modes
+sidebar_position: 20
+---
 
+Macrometa integrations support two modes of data ingestion: LOG_BASED and FULL_TABLE. This page explains each mode, as well as why you might choose one or the other.
 
+## LOG_BASED Ingestion
 
+This method uses the log files that the database generates as part of its normal operation. Whenever a change is made to the database, an entry is added to the log file. By reading this log file, it's possible to see exactly what changes have been made and when they were made.
 
-LOG_BASED and FULL_TABLE
+LOG_BASED replication works by continuously checking these log files for new entries, and then copying these changes over to the target system. This allows for near real-time replication, and because it only requires copying over the changes, it can be less resource-intensive than FULL_TABLE replication. However, not all databases support this type of replication, and it often requires special configuration settings on the database side.
 
-Include ingestion options here?
+## FULL_TABLE Ingestion
 
-  - Data Ingestion options - DFP - Can these be changed later? Saw these when setting up source at collection and when setting up target integration
-    - Batch Size
-    - Batch Flush Interval (Seconds)
-    - Batch Flush Minimum Time Gap (Seconds)
\ No newline at end of file
+This method works by scanning the entire table from the source database and copying all rows into the destination system. This happens each time the replication is run.
+
+FULL_TABLE is the most comprehensive way of replicating data, as it ensures that the destination system contains a complete copy of the source table at the point the replication was run. However, it can be resource-intensive, particularly for large tables, and it doesn't reflect changes in real-time. Instead, the changes are only visible after the next replication run.
```

Comment:
@diana-macrometa - `Instead, the changes are only visible after the next replication run`. For a given collection `FULL_TABLE` can be executed only once... So, I'm wondering whether this statement is correct.

Reply:
...I did not know that. So FULL_TABLE is a one-time thing? It won't capture ongoing changes?

I figured it would repeat on a timer or something.

Reply:
@diana-macrometa - Correct, `FULL_TABLE` is a one-time thing, and it won't capture ongoing changes. However, with `LOG_BASED` it will initially run a `FULL_TABLE` and then on top of that, use change logs to keep it in sync.

Reply:
Updated, please take a look.